### PR TITLE
a11y: Add aria-label to node

### LINKED ISSUE #2009: Test bug for Bot emulator composer
## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description

Add aria-label to node, remove `data-name` & `name` from svg.
image

## Task Item

fixes #2009

### CODE REVIEW

File: Composer/packages/extensions/visual-designer/src/components/menus/NodeMenu.tsx [hunk 1]
```diff
@@ -62,7 +61,7 @@ export const NodeMenu = ({ id, onEvent }) => {
         menuItems={menuItems}
         menuWidth={100}
         nodeSelected={nodeSelected}
-        label={formatMessage('node menu')}
+        label={formatMessage('node')}
```
Comment (corinagum) [inline Composer/packages/extensions/visual-designer/src/components/menus/NodeMenu.tsx:64]: This should be 'Node menu' or 'menu'

File: Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx [hunk 2]
```diff
@@ -96,6 +96,7 @@ export const IconMenu: React.FC<IconMenuProps> = ({
     <OverflowSet
       // @ts-ignore
       styles={{ position: 'absolute', top: 0 }}
+      aria-lable="icon menu"
```
Comment (corinagum) [inline Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx:99]: `aria-label` - spelling
I recommend `aria-label="menu"

The screen reader will say, 'Menu button'

File: Composer/packages/extensions/visual-designer/src/components/menus/NodeMenu.tsx (hunk 1 above)
Comment (alanlong9278) [reply Composer/packages/extensions/visual-designer/src/components/menus/NodeMenu.tsx:64]: Sure.

File: Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx (hunk 2 above)
Comment (alanlong9278) [reply Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx:99]: Sure,

**Pull request merged**